rules/falco_rules.yaml

```
@@ -1253,6 +1253,7 @@
     and not mcafee_writing_cma_d
     and not avinetworks_supervisor_writing_ssh
     and not multipath_writing_conf
+    and not calico-node
```

```suggestion
    and not calico_node
```
This should be the correct macro name

---

@leogr Thanks !! This has been updated now !!